lib/fungus_toast/games/game.ex

```diff
@@ -2,28 +2,46 @@ defmodule FungusToast.Games.Game do
   use Ecto.Schema
   import Ecto.Changeset
 
-  @default_status "Not Started"
-  @statuses ["Started", "Finished", "Abandoned", "Archived"]
+  @status_not_started "Not Started"
+  def status_not_started, do: @status_not_started
+
+  @status_started "Started"
+  def status_started, do: @status_started
+
+  @status_finished "Finished"
+  def status_finished, do: @status_finished
+
+  @status_abandoned "Abandoned"
+  def status_abandoned, do: @status_abandoned
+
+  @status_archived "Archived"
+  def status_archived, do: @status_archived
+
+  @statuses [@status_not_started, @status_started, @status_finished, @status_abandoned, @status_archived]
```

Comment:
Is there a new concept here that could be extracted into its own module?  Maybe call it `Game.Status` ?

Reply:
Yeah, anything to break up the Games.ex god class is a win.